### ISSUE TITLE
Possible bug in require-valid-css-selector-in-test-helpers rule

### ISSUE THREAD

**Issue by fusion2004** [ember-cli/eslint-plugin-ember#1049]
If there is a comma in the value of an attribute, it seems to get confused. Here's the example I ran into:

```javascript
assert.dom('[data-test-row="London, England, GB"] [data-test-flag] .flag-gb').exists();
```

which gave me the following error:

```
Syntax error, you used an unclosed attribute selector: "[data-test-row="London, England, GB"] [data-test-flag] .flag-gb", should be: "[data-test-row="London, England, GB"] [data-test-flag] .flag-gb]"
```

A simpler example I made up seems to have the same issue:
```javascript
assert.dom('[data-test-id="123,"]').exists();
```

**Comment by fusion2004** [comment]
Thanks, @bmish!
